dashboard/static/js/node.js

```diff
@@ -0,0 +1,293 @@
+
+                                        return true;
+                                    } else {
+                                        L.Common.showErrorTip("提示", result.msg || "编辑节点发生错误");
+                                        return false;
+                                    }
+                                },
+                                error: function () {
+                                    L.Common.showErrorTip("提示", "编辑节点请求发生异常");
+                                    return false;
+                                }
+                            });
+                        }
+                    }]
+                });
+                d.show();
+            });
+
+
+            $("#sync-node-btn").click(function () {
+                var content = $("#sync-node-tpl").html()
+                var d = dialog({
+                    title: '同步 orange 节点',
+                    width: 480,
+                    content: content,
+                    modal: true,
+                    button: [{
+                        value: '取消'
+                    }, {
+                        value: '确定',
+                        autofocus: false,
+                        callback: function () {
+
+                            $.ajax({
+                                url: '/admin/node/sync',
+                                type: 'post',
+                                cache: false,
+                                data: {},
+                                dataType: 'json',
+                                success: function (result) {
+                                    L.Common.showTipDialog("提示", result.msg);
+                                },
+                                error: function () {
+                                    L.Common.showErrorTip("提示", "添加用户请求发生异常");
+                                    return false;
+                                }
+                            });
+
+                        }
+                    }]
+                });
+                d.show();
+            });
+        },
+
+
+        loadNodes: function () {
+            $.ajax({
+                url: '/admin/nodes',
+                type: 'get',
+                cache: false,
+                data: {},
+                dataType: 'json',
+                success: function (result) {
+                    if (result.success) {
+                        _this.renderTable(result.data); //渲染table
+                        _this.data.nodes = result.data.nodes; //重新设置数据
+
+                    } else {
+                        L.Common.showTipDialog("错误提示", "查询节点请求发生错误");
+                    }
+                },
+                error: function () {
+                    L.Common.showTipDialog("提示", "查询节点请求发生异常");
+                }
+            });
+        },
+
+        renderTable: function (data, highlight_id) {
+            console.log(data, highlight_id);
```

Comment:
The `console.log` should be removed. It's a debug code. We shouldn't let the debug code be released it production.